base/components/PropControl.jsx

```diff
@@ -192,6 +194,40 @@ const PropControl = ({className, ...props}) => {
 	// Hack: Checkbox has a different html layout :( -- handled below
 	const isCheck = PropControl.KControlType.ischeckbox(type); // || PropControl.KControlType.isradio(type);
 
+	const isPublished = () => {
```

Comment:
This would be a good place for comment for future devs e.g. 

// returns false if this is a DataItem where the draft and published values are different.